### PR TITLE
Update spec for `@see` tag

### PR DESCRIPTION
**Pull request by octogonz**

### CODE REVIEW

File: tsdoc/src/details/StandardTags.ts [hunk 1]
```diff
@@ -331,26 +331,42 @@ export class StandardTags {
   /**
    * (Extended)
    *
-   * Used to document another symbol or resource that may be related to the current item being documented.
+   * Used build a list of references to an API item or other resource that may be related to the
```
Comment (hbergren) [inline tsdoc/src/details/StandardTags.ts:334]:
```suggestion
   * Used to build a list of references to an API item or other resource that may be related to the
```

File: tsdoc/src/details/StandardTags.ts [hunk 2]
```diff
@@ -331,26 +331,42 @@ export class StandardTags {
...
    *  &#42;/
-   * function foo() {}
-
-   * // Use the inline {@link} tag to include a link within a free-form description.
-   * /**
-   *  * @see {@link foo} for further information.
-   *  * @see {@link http://github.com|GitHub}
-   *  &#42;/
-   * function bar() {}
+   * function parseURL(url: string): ParsedUrl;
```
Comment (hbergren) [inline tsdoc/src/details/StandardTags.ts:351]: This is a _way_ better example!

File: tsdoc/src/details/StandardTags.ts [hunk 3]
```diff
@@ -331,26 +331,42 @@ export class StandardTags {
...
+   * Parses a string containing a Uniform Resource Locator (URL).
+   *
+   * ## See Also
+   * - ParsedUrl for the returned data structure
+   * - RFC 1738 for syntax
+   * - your developer SDK for code samples
    * ```
+   *
+   * NOTE: JSDoc attempts to automatically hyperlink the text immediately after `@see`.  Because this is ambiguous
+   * with plain text, TSDoc instead requires an explicit `{@link}` tag to make hyperlinks.
```
Comment (hbergren) [inline tsdoc/src/details/StandardTags.ts:369]: Great explanation of the `@see` tag and the note about how JSDoc's implementation is a little odd, and how TSDoc's implementation differs.

File: playground/src/DocHtmlView.tsx [hunk 4]
```diff
@@ -72,6 +72,43 @@ export class DocHtmlView extends React.Component<IDocHtmlViewProps> {
...
+
+      outputElements.push(
+        <React.Fragment key='seeAlso'>
+          <h2 className='doc-heading'>See Also</h2>
+          <ul>
+            {listItems}
+          </ul>
+        </React.Fragment>
+      );
+    }
```
Comment (hbergren) [inline playground/src/DocHtmlView.tsx:110]: Just want to make sure I understand this correctly.

So this adds UI for `@example` tags, which could potentially be a standalone PR, but it also adds UI for `@see` tags, and both of those tags are going to be rendered together in something like this:

```
Example Heading:
``` my example content ```

See Also:
- My first @see link
- My second @see link
```

Is that correct?

**Comment by hbergren** [comment]
@octogonz , just wanted to follow up with you and make sure I understood your changes correctly.

File: playground/src/DocHtmlView.tsx (hunk 4 above)
Comment (octogonz) [reply playground/src/DocHtmlView.tsx:110]: Yes, I noticed that `@example` was not being rendered by the TSDoc Playground, so I implemented support for it.  It is unrelated (but trivial enough not to put in its own PR).  The Playground is not meant to be a spec, just a suggested way that TSDoc can be rendered.

If you want to test the playground, it's pretty easy:
```bash
$ git clone https://github.com/microsoft/tsdoc.git
$ cd tsdoc/playground
$ npm install -g @microsoft/rush
$ rush install
$ rush build
$ npm run start
```

Here's a screenshot with `@example` and `@see` mixed together:

Capture
